python/lsst/jointcal/jointcal.py

```diff
@@ -189,8 +222,9 @@ def run(self, dataRefs):
             prof = pstats.Stats('jointcal_load_catalog.prof')
             prof.strip_dirs().sort_stats('cumtime').print_stats(20)
         else:
+            old_wcss = []
             for dataRef in dataRefs:
-                self._build_ccdImage(dataRef, associations, jointcalControl)
+                old_wcss.append(self._build_ccdImage(dataRef, associations, jointcalControl))
```

Comment:
Prefer a listcomp (clearer, often shorter, more pythonic):

    old_wcss = [self._build_ccdImage(dataRef, associations, jointcalControl) for dataRef in dataRefs]